packages/style-engine/src/styles/typography/index.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Internal dependencies
+ */
+import type { Style, StyleOptions } from '../../types';
+import { generateRule } from '../utils';
+
+const fontFamily = {
```

Comment:
Apologies, if this is a naive question — I haven't looked it up yet 😅 — do we need the font family support to output inline styles like this, or is it always implemented by classname? (Just wondering if this support is needed here)

Reply:
You're right to question this. I wasn't sure either.

I added it based on the backend implementation, which does check for a custom font-family in the block attribute style object.

Here's the block supports typography.php [line after the style engine migration](https://github.com/WordPress/gutenberg/blob/ef1f44994ecf68f4466ff4d8b5ab8c2f768ed567/lib/block-supports/typography.php#L110) and [before the migration](https://github.com/WordPress/gutenberg/blob/6de16981c7fef3db7cc4a9d6cfcb0ad85c916c62/lib/block-supports/typography.php#L108).

So I thought to myself, _"maybe there's a case for a custom font family of which I'm not aware"._ Then I stopped thinking. 😄 

But the [editor/JS block supports hook explicitly checks only for the attributes.fontFamily](https://github.com/WordPress/gutenberg/blob/6de16981c7fef3db7cc4a9d6cfcb0ad85c916c62/packages/block-editor/src/hooks/font-family.js#L118) value, which seems to assume that it's always a preset/classname thing.

I'll take it out to keep it inline with the current JS implementation.

Thank you!